source/universal/components/Dashboard/DashboardContent.js

```diff
@@ -40,13 +49,17 @@ function indexTags(tags) {
   return colorMap;
 }
 
-export default ({stars, tags}) => {
+export default ({addTag, stars, tags}) => {
 
   const colorMap = indexTags(tags);
 
   return (
     <div className='dashboard'>
       <div className="dashboard__tags">
+        <div onClick={() => {
+          const name = prompt('enter the tag name');
+          if (name !== '') addTag(name);
+        }}><AddTag></AddTag></div>
```

Comment:
In order to make `AddTag` more modular, we should put event handling inside the `AddTag` component:

```jsx
<AddTag onClick={addTag}></AddTag>
```

```jsx
const AddTag = ({ onClick }) => {
  return (
    <div className="tag" onClick={onClick}>
      <div className="tag__btn"><PlusIcon/></div>
    </div>
  );
};
```